instructions.md

```diff
@@ -0,0 +1,65 @@
+# instructions
```

Comment:
Capitalize to "Instructions", maybe use "How to install" or "How to get started"?